test/test_eos_tabulated.cpp

```diff
@@ -373,6 +373,8 @@ SCENARIO("SpinerEOS and EOSPAC Serialization",
           REQUIRE(isClose(P_eospac, P_spiner_orig));
           REQUIRE(isClose(P_eospac, P_spiner_rhoT));
           REQUIRE(isClose(P_eospac, P_spiner_rhoSie));
+          REQUIRE(isClose(eos_rhoT.MinInternalEnergyFromDensity(rho_trial),
+                          eos_rhoSie.MinInternalEnergyFromDensity(rho_trial)));
```

Comment:
Now that I think about this test more, it is testing whether or not we can pack up EOS data (i.e. serialization) and move it around, e.g. for MPI shared memory. I think it's appropriate to expand the test to ensure that the `MinInternalEnergyFromDensity` serializes correctly for all of these EOS.

I think it's within the scope of the MR since you're already testing the serialized spiner EOS already, and the original should "just work".

I'd probably try something like the following:
```suggestion
          const Real ecold_eospac = eos_eospac.MinInternalEnergyFromDensity(rho_trial);
          const Real ecold_spiner_orig = rhoT_orig.MinInternalEnergyFromDensity(rho_trial);
          const Real ecold_spiner_rhoT = eos_rhoT.MinInternalEnergyFromDensity(rho_trial);
          const Real ecold_spiner_rhoSie = eos_rhoSie.MinInternalEnergyFromDensity(rho_trial);
          REQUIRE(isClose(ecold_eospac, ecold_spiner_orig));
          REQUIRE(isClose(ecold_eospac, ecold_spiner_rhoT));
          REQUIRE(isClose(ecold_eospac, ecold_spiner_rhoSie));
```